### PR TITLE
add firing measurement modified event

### PR DESCRIPTION
**Pull request by sisobus**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

issue #774 
This is fixed with #786 in version 2.4.x. But not yet in version 3.
I fix this issue in version3 because i need this feature. by reference to #786.

* **What is the current behavior?** (You can also link to an open issue here)

The freehand tool doesn't fire the cornerstonetoolsmeasurementmodified event as the other measurement tools

* **What is the new behavior (if this is a feature change)?**

The event "cornerstonetoolsmeasurementmodified" be fired and the handler is called.

### CODE REVIEW

File: src/tools/annotation/FreehandMouseTool.js [hunk 1]
```diff
@@ -980,6 +992,8 @@ export default class FreehandMouseTool extends BaseAnnotationTool {
     }
 
     external.cornerstone.updateImage(element);
+
+    this.fireModifiedEvent(element, data);
```
Comment (JamesAPetts) [inline src/tools/annotation/FreehandMouseTool.js]: Thanks for adding these event fires, looks good to me.

File: src/tools/annotation/FreehandMouseTool.js [hunk 2]
```diff
@@ -810,6 +813,13 @@ export default class FreehandMouseTool extends BaseAnnotationTool {
     external.cornerstone.updateImage(eventData.element);
   }
 
+  /**
+   *
+   */
+  _measurementModifyCallback(evt) {
```
Comment (JamesAPetts) [inline src/tools/annotation/FreehandMouseTool.js]: Adding a callback and adding an internal `MEASUREMENT_MODIFIED` event listener throughout this code would only make sense if the callback does something. If you intend to add some functionality to this callback. I'm happy to wait and re-review.

However, if its just being left here as placeholder I see no benefit in adding the callback and associated `addEventListener`/`removeEventListener` commands at this time.

File: src/tools/annotation/FreehandMouseTool.js (hunk 1 above)
Comment (sisobus) [reply src/tools/annotation/FreehandMouseTool.js]: Thanks.

File: src/tools/annotation/FreehandMouseTool.js (hunk 2 above)
Comment (sisobus) [reply src/tools/annotation/FreehandMouseTool.js]: I think so.

So I will delete this callback and EventListeners. thanks for review.

**Comment by sisobus** [comment]
I think it is done. Could you review this?

**Comment by dannyrb** [comment]
@JamesAPetts, I think I've resolved the CI issue. Please feel free to merge this if it has your green light.

**Pull request merged**